test/cli/saveSequence.js

```diff
@@ -0,0 +1,38 @@
+require('../../src/ImageSequencer');
+sequencer = ImageSequencer({ ui: true });
+const saveSequence = require('../../src/cli/saveSequence.js');
+const test = require('tape');
+const { Command } = require('commander');
+
+
+test('testing save sequence function', function (t) {
+  try {
+    let program = new Command();
+    program
+      .option('--save-sequence [string]', 'Name space separated with Stringified sequence');
+
+    program.parse(['node', 'test', '--save-sequence', '"invert brightness"']);
+
+    if (program.saveSequence)
+      saveSequence(program, sequencer);
+    t.true(1, 'creation success');
```

Comment:
Instead of checking for an error, can we check if the required output was produced? Like, check if the new sequence is in the list of sequences?

Reply:
Check by reading the json file? 

Since it's using saveSequence from core `image-sequencer` under the hood

https://github.com/publiclab/image-sequencer/blob/14f13c300bf8ed360ea142322e00f02371ca9b49/src/cli/saveSequence.js#L4

https://github.com/publiclab/image-sequencer/blob/14f13c300bf8ed360ea142322e00f02371ca9b49/src/ImageSequencer.js#L352

wouldn't it be just validated if the test for that module passes?

Reply:
I am not exactly sure... You need not read the JSON file, the sequencer object also exports the saved sequences.

Reply:
Is this worth waiting or do you think we could do this as a follow-up refinement? Thank you!

Reply:
Hmm... ok? I still haven't understood what this test is supposed to check :sweat_smile:

Reply:
I believe the initial implementation is to verify if the arguement are parsed the right way or not. Like #1634 changes how the arguements are passed down to it.

Reply:
Ok, sure!